tutorials/publish_subscribe/java/build.gradle

```diff
@@ -0,0 +1,49 @@
+plugins {
+    id 'application'
+}
+
+dependencies {
+    implementation files(System.getenv('NDDSHOME') + '/lib/java/nddsjava.jar')
+}
+
+sourceSets {
+    main {
+        java {
+            srcDirs = ['.']
+        }
+    }
+}
+
+task generateCode(type: Exec) {
+    commandLine System.getenv('NDDSHOME') + '/bin/rtiddsgen',
+        '-language', 'Java',
+        'home_automation.idl'
+    onlyIf {
```

Comment:
so this will only trigger if the generated .java files don't exist? Can we make it a dependency such that if home_automation.idl has been modified more recently we re-generate?